lib/matobj.gi

```diff
@@ -1568,3 +1568,572 @@ InstallMethod( \[\,\]\:\=, "for a matrix object, two positions, and an object",
   end );
 
 
+############################################################################
+# Elementary matrix operations
+############################################################################
+
+############################################################################
+##
+#O  MultMatrixRow( <M>, <row>, <scalar> )
```

Comment:
scalar -> elm

Reply:
In fact M,row, scalar -> mat,i,elm.

or we just delete these lines, they have no purpose?

Reply:
Sergio told us that those lines are legacy and not needed any more. We‘ll go through our code and delete those everywhere.